### ISSUE TITLE
Unable to press any key with TalkBack activated

### ISSUE THREAD

**Issue by mgoral** [AnySoftKeyboard/AnySoftKeyboard#2316]
Hi,

TalkBack is Android accessibility technology and screen reader for visually impaired people which is freely available e.g. from F-Droid Store and Google Play Store. When it is enabled I'm unable to press any key on AnySoftKeyboard. Other keyboards, e.g. AOSP keyboard work fine.

I assume that this is somehow related to the gestures because when I'm sliding with two fingers over the keyboard sometimes letters are randomly chosen and alternative letters are activated, like during long-press. When gesture mode is changed to beta "swype-to-type", letters are also sometimes selected, but alternative letters are no longer displayed and overall experience changes a little bit.

Once I was able to type something by clicking multiple times on a single key or by clicking once on it and then twice outside of the keyboard, but after changing to AOSP and back to ASK this never happened again.

This happens on all themes and on different keyboard layouts.

### Steps to reproduce
1. Install TalkBack
2. Enable TalkBack from Settings -> Accessibility -> TalkBack
3. Select any input field (TalkBack gestures to select different controls shown on the screen: left-to-right, right-to-left, up-to-down and down-to-up; you can also click any input field as usually; when it's finally selected, double click anywhere on the screen to activate it)
4. Try typing anything. Normally first click selects the letter and second one types it.
5. Compare it to experience of AOSP keyboard where user can swipe his finger over the whole keyboard and during that letters are selected and read by TTS engine. They can be also typed almost normally e.g. by people with partial vision, but with a noticable lag.

### Expected behaviour
People should be able to use ASK when TalkBack is enabled

**Android OS version:**
Lineage OS with microG 15.1-20190313-microG-tissot
Android 8.1.0

**Device manufacturer and model:**
Xiaomi Mi A1 (tissot)

**List of installed add-ons (like languages, or themes):**
- Polish Language Pack

**Comment by mh-nichts** [comment]
I confirm, this is a serious accessibility issue.
I have it on my phone as well.

Android OS version:
Android 10

Device manufacturer and model:
Nokia 7.1 (TA-1095)

List of installed add-ons (like languages, or themes):
French language pack

**Comment by ObjectInSpace** [comment]
There is accessibility support in the AOSP keyboard but it is not present in this fork. SimpleKeyboard has the same problem. AFAIK all that needs to be done is to merge https://android.googlesource.com/platform/packages/inputmethods/LatinIME/+/refs/heads/master/java/src/com/android/inputmethod/accessibility?autodive=0%2F%2F/

Just to be clear, the keyboard is completely broken at present for all blind people, and has been for years. Are there any plans to do anything about this, or should we all use OpenBoard?
